rmgpy/molecule/parser/inchiTest.py

```diff
@@ -0,0 +1,92 @@
+import unittest
+
+from rmgpy.molecule.parser.inchi import AugmentedInChI, InChI, ignore_prefix, compose_aug_inchi, INCHI_PREFIX, U_LAYER_PREFIX
+
+
+class InChITest(unittest.TestCase):
+
+    def test_constructor(self):
+        inchi1 = InChI('InChI=1S/foo')
+        assert inchi1 is not None
```

Comment:
I thought unit tests should use `self.assert` (or `self.assertTrue`, `self.assertEqual`, etc. etc.) rather than `assert` so that they're recognized as test failures rather than errors.